en/1-who-we-are.md

```diff
@@ -1,14 +1,12 @@
 
-# 1. Who we are – the Meeds Association
+# 1. About Us
 
-The Meeds association is a non-profit organization originally launched by several Open Source Software (OSS) communities around the common vision of a free digital world – a decentralized world that would not be controlled by a handful of centralized mega-corporations.
+Meeds is a collaborative effort led by diverse Open Source Software (OSS) communities, dedicated to creating a decentralized digital world that resists centralized corporate control and fosters a free and open environment.
 
-The Meeds association is incorporated in Geneva, as a non-profit association under the Swiss law.
+The legal entity behind Meeds is the _Meeds Association_, a non-profit organization incorporated under Swiss law in Geneva. It is responsible for regulating the issuance of the Meeds(MEED) token, an ERC20 token recognized as a utility token.
 
-The association regulates the issuance of the Meeds currency, an ERC20 token, recognized as a utility token.
+Functioning as a Decentralized Autonomous Organization (DAO), Meeds emphasizes democratic governance. Token holders with governance rights participate in key decisions, including annual board elections, ensuring transparency and community involvement in organizational changes.
```

Comment:
```suggestion
Functioning as a Decentralized Autonomous Organization (DAO), Meeds emphasizes on democratic governance, where token holders with governance rights participate in key decision making, such as, annual board elections, ensuring transparency and community involvement in organizational changes.
```

Reply:
The word "including" in the sentence means that the writer had mentioned some key decisions before the annual board elections, which in this case, was not mentioned. since the only key decision mentioned here is annual board elections, then for more clarity, I believe that the phrase "such as" should be used instead or "including"

Reply:
✅ Approved!